src/hooks/useGetDelegationActions.ts

```diff
@@ -53,9 +61,9 @@ export function useGetDelegationActions(delegation: Delegation | CompactDelegati
   }
 
   if (
-    delegation.kind === 'DELEGATED_PRODUCER' &&
     delegation.state === 'WAITING_FOR_APPROVAL' &&
-    delegation.delegate.id === jwt?.organizationId
+    delegation.delegate.id === jwt?.organizationId &&
+    isAdmin
```

Comment:
Let's make this if more obvious to read. I would extract `delegation.delegate.id === jwt?.organizationId` into a `isActualTenantDelegate` (or something like that) constant